k8s/helm/Chart.yaml

```
@@ -12,4 +12,4 @@ dependencies:
 annotations:
   # For k8s-loki it is important to include "prerelease"-versions like "2.9.1-2", so there must be a "-0" after the patch-version.
   # see https://helm.sh/docs/chart_best_practices/dependencies/#prerelease-versions for more information
-  "k8s.cloudogu.com/ces-dependency/k8s-loki": "^2.9.1-0"
\ No newline at end of file
+  "k8s.cloudogu.com/ces-dependency/k8s-loki": ">=2.9.1-0"
```

As I said in https://github.com/cloudogu/k8s-ces-control/pull/40, I'd prefer it like this:
```suggestion
  "k8s.cloudogu.com/ces-dependency/k8s-loki": "^2.9.1-0, ^3.0.0-0"
```